README.md

```diff
@@ -1,22 +1,70 @@
-ibc-solidity-contract is the EVM-compatible IBC implementation for chain based on [AXON](https://github.com/axonweb3/axon).
+# IBC Solidity For Forcerelay
 
-## Contract Compilation
+The IBC solidity contracts for [Axon](https://github.com/axonweb3/axon) is based on [yui-ibc-solidity](https://github.com/hyperledger-labs/yui-ibc-solidity). Most of the instructions, such as registering IBC modules and light client, remain the same.
+
+NOTE: This is pre-beta non-production-quality software, not ready for use in a production environment.
+
+## Limitation
+
+`yui-ibc-solidity` is designed to be compatible with the IBC core protocol, providing minimal compatibility. In contrast, `ibc-solidity-contract` goes further by providing additional APIs to support Forcerelay's query and storage demands. Consequently, the contract size exceeds the limits imposed by standard EVM-compatible chains, such as Arbitrum, Avalanche, BSC. As a result, this project can only be deployed on Axon, taking advantage of Axon's ability to transcend the EVM's constraints.
+
+## Installation and Testing
+
+First, ensure that, Node.js and Yarn have been already installed:
 
 ```bash
-# 1. Installs dependencies
-yarn install
+$ git clone https://github.com/synapseweb3/ibc-solidity-contract
+$ cd ibc-solidity-contract
+$ yarn install
+$ yarn compile
+$ yarn test
+```
+
+## Deploy on Axon
```

Comment:
Do you need to add the operation of axon to deploy self-built nodes and the corresponding ci process?

Reply:
> Do you need to add the operation of axon to deploy self-built nodes and the corresponding ci process?

It is ok at the current stage.
AXON_HTTP_RPC_URL should be configured in the `.env` file, mentioned in the docs:
https://github.com/synapseweb3/ibc-solidity-contract/blob/b6a93f83f282eef31f9517dcd186099a0fa489ba/README.md?plain=1#L25-L30